Link on the Next steps is not working

laraaleite opened the pull request:
The link, in the Next steps section, to the PHP Develop Center is not working.
https://docs.microsoft.com/en-us/develop/php/ > The page you requested could not be found

azurecla commented:
Hi __@danmarshall__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ryancrawcour commented:
The steps result in the same outcome as before. adding the view from within the controller is just another way of doing it, with slightly fewer clicks. so, sure, I don't see an issue with this as long as we're consistent throughout the remainder of the tutorial when adding views. 

This change also removes a screenshot of the Solution Explorer flyout menu, but doesn't include a screenshot of clicking inside the code editor. Is this self explanatory enough for users that have never done it this way before to figure out? Do we need a new screenshot here?

danmarshall commented:
Yes I see your point about consistency. Instead of making all the other changes I will retract this pull request. Thanks!

laraaleite committed:
Link on the Next steps is not working

The link, in the Next steps section, to the PHP Develop Center is not working.
https://docs.microsoft.com/en-us/develop/php/ > The page you requested could not be found
